be/src/http/http_common.h

```diff
@@ -60,4 +62,118 @@ static const std::string HTTP_TWO_PHASE_COMMIT = "two_phase_commit";
 static const std::string HTTP_TXN_ID_KEY = "txn_id";
 static const std::string HTTP_TXN_OPERATION_KEY = "txn_operation";
 
+static const std::unordered_set<std::string> kAvailableStreamLoadRequestFields {
+        // below are avaiable request fields defined in stream load
+        HTTP_LABEL_KEY,
+        HTTP_TWO_PHASE_COMMIT,
+        HTTP_COMPRESS_TYPE,
+        HTTP_FORMAT_KEY,
+        HTTP_FORMAT_KEY,
+        HTTP_READ_JSON_BY_LINE,
+        HTTP_TIMEOUT,
+        HTTP_COMMENT,
+        HTTP_COLUMNS,
+        HTTP_WHERE,
+        HTTP_COLUMN_SEPARATOR,
+        HTTP_LINE_DELIMITER,
+        HTTP_PARTITIONS,
+        HTTP_TEMP_PARTITIONS,
+        HTTP_NEGATIVE,
+        HTTP_STRICT_MODE,
+        HTTP_TIMEZONE,
+        HTTP_EXEC_MEM_LIMIT,
+        HTTP_JSONPATHS,
+        HTTP_JSONROOT,
+        HTTP_STRIP_OUTER_ARRAY,
+        HTTP_NUM_AS_STRING,
+        HTTP_FUZZY_PARSE,
+        HTTP_READ_JSON_BY_LINE,
+        HTTP_FUNCTION_COLUMN + "." + HTTP_SEQUENCE_COL,
+        HTTP_SEND_BATCH_PARALLELISM,
+        HTTP_LOAD_TO_SINGLE_TABLET,
+        HTTP_MERGE_TYPE,
+        HTTP_DELETE_CONDITION,
+        HTTP_MAX_FILTER_RATIO,
+        HTTP_HIDDEN_COLUMNS,
+        HTTP_TRIM_DOUBLE_QUOTES,
+        HTTP_SKIP_LINES,
+        HTTP_ENABLE_PROFILE,
+        HTTP_PARTIAL_COLUMNS,
+
+        // below are http standard request fields
```

Comment:
why do we need check the http headers? If http version upgrade or some headers we do not know currently, we need fix doris's code.

Reply:
user is easy to misuse the new added partial_columns header, if it's not set correctly, the load will still success, but result would not match the user's expectation.